src/librustc_feature/active.rs

```diff
@@ -532,6 +532,9 @@ declare_features! (
     /// Allows using `&mut` in constant functions.
     (active, const_mut_refs, "1.41.0", Some(57349), None),
 
+    // Allows limiting the evaluation steps of const expressions
+    (active, const_eval_limit, "1.41.0", Some(67217), None),
```

Comment:
This version is wrong and should be bumped to 1.43.0.

Reply:
The PR is also closing the issue this is pointing into.

Reply:
Which issue should this point to? There is no tracking issue I am aware of.

Reply:
He means that you should remove the "closes" line from your PR description.

Reply:
Ah okay, thanks!